openstreetmap example error

Hello, Using the baremaps postgis docker container and importing the openstreet map example workflow, I am getting the following error from the tile server when running serving tiles.

```
[ERROR] 2023-01-28 20:29:21.942 [servicetalk-global-executor-1-6] DevResources - Tile error
org.apache.baremaps.database.tile.TileStoreException: org.postgresql.util.PSQLException: ERROR: syntax error at end of input
  Position: 7
	at org.apache.baremaps.database.tile.PostgresTileStore.read(PostgresTileStore.java:122) ~[baremaps-core-0.7.1-SNAPSHOT.jar:?]
	at org.apache.baremaps.server.DevResources.getTile(DevResources.java:137) ~[baremaps-server-0.7.1-SNAPSHOT.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory.lambda$static$0(ResourceMethodInvocationHandlerFactory.java:52) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:124) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:167) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$ResponseOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:176) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:79) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:475) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:397) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:81) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:255) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:248) ~[jersey-common-2.35.jar:?]
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:244) ~[jersey-common-2.35.jar:?]
	at org.glassfish.jersey.internal.Errors.process(Errors.java:292) ~[jersey-common-2.35.jar:?]
	at org.glassfish.jersey.internal.Errors.process(Errors.java:274) ~[jersey-common-2.35.jar:?]
	at org.glassfish.jersey.internal.Errors.process(Errors.java:244) ~[jersey-common-2.35.jar:?]
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:265) ~[jersey-common-2.35.jar:?]
	at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:234) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:684) ~[jersey-server-2.35.jar:?]
	at io.servicetalk.http.router.jersey.DefaultJerseyStreamingHttpRouter.handle0(DefaultJerseyStreamingHttpRouter.java:273) ~[servicetalk-http-router-jersey-0.42.18.jar:0.42.18]
	at io.servicetalk.http.router.jersey.DefaultJerseyStreamingHttpRouter.access$100(DefaultJerseyStreamingHttpRouter.java:79) ~[servicetalk-http-router-jersey-0.42.18.jar:0.42.18]
	at io.servicetalk.http.router.jersey.DefaultJerseyStreamingHttpRouter$3.handleSubscribe(DefaultJerseyStreamingHttpRouter.java:204) ~[servicetalk-http-router-jersey-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Single.handleSubscribe(Single.java:2649) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Single.delegateSubscribe(Single.java:2620) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.SingleFlatMapCompletable.handleSubscribe(SingleFlatMapCompletable.java:43) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Completable.lambda$subscribeWithContext$5(Completable.java:2242) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.ContextPreservingRunnable.run(ContextPreservingRunnable.java:46) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Completable.subscribeWithContext(Completable.java:2242) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Completable.subscribeInternal(Completable.java:1732) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.SourceToFuture$CompletableToFuture.createAndSubscribe(SourceToFuture.java:162) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Completable.toFuture(Completable.java:1705) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.http.api.StreamingHttpServiceToBlockingStreamingHttpService.handle(StreamingHttpServiceToBlockingStreamingHttpService.java:57) ~[servicetalk-http-api-0.42.18.jar:0.42.18]
	at io.servicetalk.http.api.BlockingStreamingToStreamingService$1.handleSubscribe(BlockingStreamingToStreamingService.java:130) ~[servicetalk-http-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Single.handleSubscribe(Single.java:2649) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.SingleShareContextOnSubscribe.handleSubscribe(SingleShareContextOnSubscribe.java:38) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Single.subscribeWithContext(Single.java:2629) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Single.subscribeAndReturnContext(Single.java:2607) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Single.subscribeInternal(Single.java:1816) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.SingleFlatMapSingle$SubscriberImpl.onSuccess(SingleFlatMapSingle.java:80) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.ContextPreservingSingleSubscriber.onSuccess(ContextPreservingSingleSubscriber.java:77) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.AbstractSubmitSingle.lambda$handleSubscribe$0(AbstractSubmitSingle.java:49) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.ContextPreservingRunnable.run(ContextPreservingRunnable.java:46) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) ~[?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: org.postgresql.util.PSQLException: ERROR: syntax error at end of input
  Position: 7
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2676) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2366) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:356) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:496) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:413) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:333) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:319) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:295) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:244) ~[postgresql-42.5.0.jar:42.5.0]
	at com.zaxxer.hikari.pool.ProxyStatement.executeQuery(ProxyStatement.java:110) ~[HikariCP-5.0.1.jar:?]
	at com.zaxxer.hikari.pool.HikariProxyStatement.executeQuery(HikariProxyStatement.java) ~[HikariCP-5.0.1.jar:?]
	at org.apache.baremaps.database.tile.PostgresTileStore.read(PostgresTileStore.java:108) ~[baremaps-core-0.7.1-SNAPSHOT.jar:?]
	... 50 more
[ERROR] 2023-01-28 20:29:21.942 [servicetalk-global-executor-1-5] DevResources - Tile error
org.apache.baremaps.database.tile.TileStoreException: org.postgresql.util.PSQLException: ERROR: syntax error at end of input
  Position: 7
	at org.apache.baremaps.database.tile.PostgresTileStore.read(PostgresTileStore.java:122) ~[baremaps-core-0.7.1-SNAPSHOT.jar:?]
	at org.apache.baremaps.server.DevResources.getTile(DevResources.java:137) ~[baremaps-server-0.7.1-SNAPSHOT.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory.lambda$static$0(ResourceMethodInvocationHandlerFactory.java:52) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:124) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:167) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$ResponseOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:176) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:79) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:475) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:397) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:81) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:255) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:248) ~[jersey-common-2.35.jar:?]
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:244) ~[jersey-common-2.35.jar:?]
	at org.glassfish.jersey.internal.Errors.process(Errors.java:292) ~[jersey-common-2.35.jar:?]
	at org.glassfish.jersey.internal.Errors.process(Errors.java:274) ~[jersey-common-2.35.jar:?]
	at org.glassfish.jersey.internal.Errors.process(Errors.java:244) ~[jersey-common-2.35.jar:?]
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:265) ~[jersey-common-2.35.jar:?]
	at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:234) ~[jersey-server-2.35.jar:?]
	at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:684) ~[jersey-server-2.35.jar:?]
	at io.servicetalk.http.router.jersey.DefaultJerseyStreamingHttpRouter.handle0(DefaultJerseyStreamingHttpRouter.java:273) ~[servicetalk-http-router-jersey-0.42.18.jar:0.42.18]
	at io.servicetalk.http.router.jersey.DefaultJerseyStreamingHttpRouter.access$100(DefaultJerseyStreamingHttpRouter.java:79) ~[servicetalk-http-router-jersey-0.42.18.jar:0.42.18]
	at io.servicetalk.http.router.jersey.DefaultJerseyStreamingHttpRouter$3.handleSubscribe(DefaultJerseyStreamingHttpRouter.java:204) ~[servicetalk-http-router-jersey-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Single.handleSubscribe(Single.java:2649) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Single.delegateSubscribe(Single.java:2620) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.SingleFlatMapCompletable.handleSubscribe(SingleFlatMapCompletable.java:43) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Completable.lambda$subscribeWithContext$5(Completable.java:2242) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.ContextPreservingRunnable.run(ContextPreservingRunnable.java:46) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Completable.subscribeWithContext(Completable.java:2242) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Completable.subscribeInternal(Completable.java:1732) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.SourceToFuture$CompletableToFuture.createAndSubscribe(SourceToFuture.java:162) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Completable.toFuture(Completable.java:1705) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.http.api.StreamingHttpServiceToBlockingStreamingHttpService.handle(StreamingHttpServiceToBlockingStreamingHttpService.java:57) ~[servicetalk-http-api-0.42.18.jar:0.42.18]
	at io.servicetalk.http.api.BlockingStreamingToStreamingService$1.handleSubscribe(BlockingStreamingToStreamingService.java:130) ~[servicetalk-http-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Single.handleSubscribe(Single.java:2649) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.SingleShareContextOnSubscribe.handleSubscribe(SingleShareContextOnSubscribe.java:38) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Single.subscribeWithContext(Single.java:2629) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Single.subscribeAndReturnContext(Single.java:2607) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.Single.subscribeInternal(Single.java:1816) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.SingleFlatMapSingle$SubscriberImpl.onSuccess(SingleFlatMapSingle.java:80) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.ContextPreservingSingleSubscriber.onSuccess(ContextPreservingSingleSubscriber.java:77) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.AbstractSubmitSingle.lambda$handleSubscribe$0(AbstractSubmitSingle.java:49) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at io.servicetalk.concurrent.api.ContextPreservingRunnable.run(ContextPreservingRunnable.java:46) ~[servicetalk-concurrent-api-0.42.18.jar:0.42.18]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) ~[?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: org.postgresql.util.PSQLException: ERROR: syntax error at end of input
  Position: 7
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2676) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2366) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:356) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:496) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:413) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:333) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:319) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:295) ~[postgresql-42.5.0.jar:42.5.0]
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:244) ~[postgresql-42.5.0.jar:42.5.0]
	at com.zaxxer.hikari.pool.ProxyStatement.executeQuery(ProxyStatement.java:110) ~[HikariCP-5.0.1.jar:?]
	at com.zaxxer.hikari.pool.HikariProxyStatement.executeQuery(HikariProxyStatement.java) ~[HikariCP-5.0.1.jar:?]
	at org.apache.baremaps.database.tile.PostgresTileStore.read(PostgresTileStore.java:108) ~[baremaps-core-0.7.1-SNAPSHOT.jar:?]
	... 50 more
```

@bchapuis, thank you, `baosystems/postgis` does seem to perform better on my m1 macbook air, and I am now rendering map tiles! 


I am still seeing the errors in the console logs.
```
[ERROR] 2023-01-29 10:15:20.101 [servicetalk-global-executor-1-3] DevResources - Tile error
org.apache.baremaps.database.tile.TileStoreException: org.postgresql.util.PSQLException: ERROR: syntax error at end of input
  Position: 7
	at org.apache.baremaps.database.tile.PostgresTileStore.read(PostgresTileStore.java:122) ~[baremaps-core-0.7.1-SNAPSHOT.jar:?]
	at org.apache.baremaps.server.DevResources.getTile(DevResources.java:137) ~[baremaps-server-0.7.1-SNAPSHOT.jar:?]
```
And in the postgres logs:

```
2023-01-29 14:15:20.094 UTC [89] ERROR:  syntax error at end of input at character 7
2023-01-29 14:15:20.094 UTC [89] STATEMENT:  with  
2023-01-29 14:15:20.094 UTC [87] ERROR:  syntax error at end of input at character 7
2023-01-29 14:15:20.094 UTC [87] STATEMENT:  with  
2023-01-29 14:15:20.094 UTC [86] ERROR:  syntax error at end of input at character 7
2023-01-29 14:15:20.094 UTC [86] STATEMENT:  with  
2023-01-29 14:19:06.779 UTC [69] LOG:  checkpoint starting: time
```
But I am scrolling outside the import zone. Perhaps the errors are related to trying to render tiles that haven't been imported?

The only exceptions I see are Netty related. 
```
[DEBUG] 2023-03-06 08:29:37.888 [servicetalk-global-io-executor-1-3] DefaultNettyConnection - [id: 0xb4c156ba, L:/[0:0:0:0:0:0:0:1%0]:9000 - R:/[0:0:0:0:0:0:0:1%0]:62798] Received a user event: io.netty.channel.socket.ChannelInputShutdownEvent@1b1eda1
[DEBUG] 2023-03-06 08:29:37.888 [servicetalk-global-io-executor-1-7] DefaultNettyConnection - [id: 0xbe516ca7, L:/[0:0:0:0:0:0:0:1%0]:9000 - R:/[0:0:0:0:0:0:0:1%0]:62803] Received a user event: io.netty.channel.socket.ChannelInputShutdownEvent@1b1eda1
[DEBUG] 2023-03-06 08:29:37.888 [servicetalk-global-io-executor-1-5] DefaultNettyConnection - [id: 0xff27e109, L:/[0:0:0:0:0:0:0:1%0]:9000 - R:/[0:0:0:0:0:0:0:1%0]:62801] Received a user event: io.netty.channel.socket.ChannelInputShutdownEvent@1b1eda1
[DEBUG] 2023-03-06 08:29:37.890 [servicetalk-global-io-executor-1-7] DefaultNettyConnection - [id: 0xbe516ca7, L:/[0:0:0:0:0:0:0:1%0]:9000 - R:/[0:0:0:0:0:0:0:1%0]:62803] Received a user event: io.netty.channel.socket.ChannelInputShutdownReadComplete@79b19a50
[DEBUG] 2023-03-06 08:29:37.891 [servicetalk-global-io-executor-1-5] DefaultNettyConnection - [id: 0xff27e109, L:/[0:0:0:0:0:0:0:1%0]:9000 - R:/[0:0:0:0:0:0:0:1%0]:62801] Received a user event: io.netty.channel.socket.ChannelInputShutdownReadComplete@79b19a50
[DEBUG] 2023-03-06 08:29:37.891 [servicetalk-global-io-executor-1-3] DefaultNettyConnection - [id: 0xb4c156ba, L:/[0:0:0:0:0:0:0:1%0]:9000 - R:/[0:0:0:0:0:0:0:1%0]:62798] Received a user event: io.netty.channel.socket.ChannelInputShutdownReadComplete@79b19a50
[DEBUG] 2023-03-06 08:29:37.891 [servicetalk-global-io-executor-1-7] DefaultNettyConnection - [id: 0xbe516ca7, L:/[0:0:0:0:0:0:0:1%0]:9000 - R:/[0:0:0:0:0:0:0:1%0]:62803] Emitted CloseEvent: CHANNEL_CLOSED_INBOUND
[DEBUG] 2023-03-06 08:29:37.891 [servicetalk-global-io-executor-1-5] DefaultNettyConnection - [id: 0xff27e109, L:/[0:0:0:0:0:0:0:1%0]:9000 - R:/[0:0:0:0:0:0:0:1%0]:62801] Emitted CloseEvent: CHANNEL_CLOSED_INBOUND
[DEBUG] 2023-03-06 08:29:37.892 [servicetalk-global-io-executor-1-3] DefaultNettyConnection - [id: 0xb4c156ba, L:/[0:0:0:0:0:0:0:1%0]:9000 - R:/[0:0:0:0:0:0:0:1%0]:62798] Emitted CloseEvent: CHANNEL_CLOSED_INBOUND
[DEBUG] 2023-03-06 08:29:41.789 [servicetalk-global-io-executor-1-7] DefaultNettyConnection - [id: 0xbe516ca7, L:/[0:0:0:0:0:0:0:1%0]:9000 - R:/[0:0:0:0:0:0:0:1%0]:62803] Received a user event: io.netty.channel.socket.ChannelOutputShutdownEvent@6745b6cc
[DEBUG] 2023-03-06 08:29:41.792 [servicetalk-global-io-executor-1-7] RequestResponseCloseHandler - [id: 0xbe516ca7, L:/[0:0:0:0:0:0:0:1%0]:9000 - R:/[0:0:0:0:0:0:0:1%0]:62803] Closing channel – evt: CHANNEL_CLOSED_OUTBOUND
[DEBUG] 2023-03-06 08:29:41.830 [servicetalk-global-io-executor-1-3] DefaultNettyConnection - [id: 0xb4c156ba, L:/[0:0:0:0:0:0:0:1%0]:9000 - R:/[0:0:0:0:0:0:0:1%0]:62798] Received a user event: io.netty.channel.socket.ChannelOutputShutdownEvent@6745b6cc
[DEBUG] 2023-03-06 08:29:41.830 [servicetalk-global-io-executor-1-5] DefaultNettyConnection - [id: 0xff27e109, L:/[0:0:0:0:0:0:0:1%0]:9000 - R:/[0:0:0:0:0:0:0:1%0]:62801] Received a user event: io.netty.channel.socket.ChannelOutputShutdownEvent@6745b6cc
[DEBUG] 2023-03-06 08:29:41.835 [servicetalk-global-io-executor-1-5] RequestResponseCloseHandler - [id: 0xff27e109, L:/[0:0:0:0:0:0:0:1%0]:9000 - R:/[0:0:0:0:0:0:0:1%0]:62801] Closing channel – evt: CHANNEL_CLOSED_OUTBOUND
[DEBUG] 2023-03-06 08:29:41.833 [servicetalk-global-io-executor-1-3] RequestResponseCloseHandler - [id: 0xb4c156ba, L:/[0:0:0:0:0:0:0:1%0]:9000 - R:/[0:0:0:0:0:0:0:1%0]:62798] Closing channel – evt: CHANNEL_CLOSED_OUTBOUND
[DEBUG] 2023-03-06 08:29:41.837 [servicetalk-global-io-executor-1-3] WriteStreamSubscriber - [id: 0xb4c156ba, L:/[0:0:0:0:0:0:0:1%0]:9000 ! R:/[0:0:0:0:0:0:0:1%0]:62798] Terminate subscriber with an error, state: 1100
io.netty.channel.socket.ChannelOutputShutdownException: Channel output shutdown
	at io.netty.channel.AbstractChannel$AbstractUnsafe.shutdownOutput(AbstractChannel.java:650) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.handleWriteError(AbstractChannel.java:953) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.flush0(AbstractChannel.java:933) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.kqueue.AbstractKQueueChannel$AbstractKQueueUnsafe.flush0(AbstractKQueueChannel.java:507) ~[netty-transport-classes-kqueue-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.flush(AbstractChannel.java:895) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.flush(DefaultChannelPipeline.java:1372) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush0(AbstractChannelHandlerContext.java:750) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush(AbstractChannelHandlerContext.java:742) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.flush(AbstractChannelHandlerContext.java:728) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.transport.netty.internal.DefaultNettyConnection$NettyToStChannelInboundHandler.channelWritabilityChanged(DefaultNettyConnection.java:815) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:441) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:428) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelWritabilityChanged(AbstractChannelHandlerContext.java:421) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelWritabilityChanged(DefaultChannelPipeline.java:1433) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:441) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:428) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelWritabilityChanged(DefaultChannelPipeline.java:931) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.fireChannelWritabilityChanged(ChannelOutboundBuffer.java:628) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.setUnwritable(ChannelOutboundBuffer.java:607) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.incrementPendingOutboundBytes(ChannelOutboundBuffer.java:177) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.addMessage(ChannelOutboundBuffer.java:129) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.write(AbstractChannel.java:882) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.write(DefaultChannelPipeline.java:1367) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:709) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:792) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:702) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:697) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.http.netty.HttpObjectEncoder.encodeChunkedContent(HttpObjectEncoder.java:402) ~[servicetalk-http-netty-0.42.18.jar:0.42.18]
	at io.servicetalk.http.netty.HttpObjectEncoder.write(HttpObjectEncoder.java:206) ~[servicetalk-http-netty-0.42.18.jar:0.42.18]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:709) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:792) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:702) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline.write(DefaultChannelPipeline.java:1015) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel.write(AbstractChannel.java:301) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber$AllWritesPromise.writeNext(WriteStreamSubscriber.java:413) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber.doWrite(WriteStreamSubscriber.java:174) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber.lambda$onNext$2(WriteStreamSubscriber.java:164) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.kqueue.KQueueEventLoop.run(KQueueEventLoop.java:295) ~[netty-transport-classes-kqueue-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: io.netty.channel.unix.Errors$NativeIoException: writevAddresses(..) failed: Broken pipe
[DEBUG] 2023-03-06 08:29:41.837 [servicetalk-global-io-executor-1-5] WriteStreamSubscriber - [id: 0xff27e109, L:/[0:0:0:0:0:0:0:1%0]:9000 ! R:/[0:0:0:0:0:0:0:1%0]:62801] Terminate subscriber with an error, state: 1100
io.netty.channel.socket.ChannelOutputShutdownException: Channel output shutdown
	at io.netty.channel.AbstractChannel$AbstractUnsafe.shutdownOutput(AbstractChannel.java:650) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.handleWriteError(AbstractChannel.java:953) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.flush0(AbstractChannel.java:933) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.kqueue.AbstractKQueueChannel$AbstractKQueueUnsafe.flush0(AbstractKQueueChannel.java:507) ~[netty-transport-classes-kqueue-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.flush(AbstractChannel.java:895) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.flush(DefaultChannelPipeline.java:1372) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush0(AbstractChannelHandlerContext.java:750) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush(AbstractChannelHandlerContext.java:742) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.flush(AbstractChannelHandlerContext.java:728) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.transport.netty.internal.DefaultNettyConnection$NettyToStChannelInboundHandler.channelWritabilityChanged(DefaultNettyConnection.java:815) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:441) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:428) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelWritabilityChanged(AbstractChannelHandlerContext.java:421) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelWritabilityChanged(DefaultChannelPipeline.java:1433) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:441) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:428) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelWritabilityChanged(DefaultChannelPipeline.java:931) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.fireChannelWritabilityChanged(ChannelOutboundBuffer.java:628) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.setUnwritable(ChannelOutboundBuffer.java:607) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.incrementPendingOutboundBytes(ChannelOutboundBuffer.java:177) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.addMessage(ChannelOutboundBuffer.java:129) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.write(AbstractChannel.java:882) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.write(DefaultChannelPipeline.java:1367) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:709) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:792) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:702) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:697) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.http.netty.HttpObjectEncoder.encodeChunkedContent(HttpObjectEncoder.java:402) ~[servicetalk-http-netty-0.42.18.jar:0.42.18]
	at io.servicetalk.http.netty.HttpObjectEncoder.write(HttpObjectEncoder.java:206) ~[servicetalk-http-netty-0.42.18.jar:0.42.18]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:709) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:792) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:702) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline.write(DefaultChannelPipeline.java:1015) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel.write(AbstractChannel.java:301) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber$AllWritesPromise.writeNext(WriteStreamSubscriber.java:413) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber.doWrite(WriteStreamSubscriber.java:174) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber.lambda$onNext$2(WriteStreamSubscriber.java:164) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.kqueue.KQueueEventLoop.run(KQueueEventLoop.java:295) ~[netty-transport-classes-kqueue-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: io.netty.channel.unix.Errors$NativeIoException: writevAddresses(..) failed: Broken pipe
[DEBUG] 2023-03-06 08:29:41.800 [servicetalk-global-io-executor-1-7] WriteStreamSubscriber - [id: 0xbe516ca7, L:/[0:0:0:0:0:0:0:1%0]:9000 ! R:/[0:0:0:0:0:0:0:1%0]:62803] Terminate subscriber with an error, state: 1100
io.netty.channel.socket.ChannelOutputShutdownException: Channel output shutdown
	at io.netty.channel.AbstractChannel$AbstractUnsafe.shutdownOutput(AbstractChannel.java:650) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.handleWriteError(AbstractChannel.java:953) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.flush0(AbstractChannel.java:933) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.kqueue.AbstractKQueueChannel$AbstractKQueueUnsafe.flush0(AbstractKQueueChannel.java:507) ~[netty-transport-classes-kqueue-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.flush(AbstractChannel.java:895) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.flush(DefaultChannelPipeline.java:1372) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush0(AbstractChannelHandlerContext.java:750) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush(AbstractChannelHandlerContext.java:742) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.flush(AbstractChannelHandlerContext.java:728) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.transport.netty.internal.DefaultNettyConnection$NettyToStChannelInboundHandler.channelWritabilityChanged(DefaultNettyConnection.java:815) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:441) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:428) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelWritabilityChanged(AbstractChannelHandlerContext.java:421) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelWritabilityChanged(DefaultChannelPipeline.java:1433) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:441) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:428) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelWritabilityChanged(DefaultChannelPipeline.java:931) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.fireChannelWritabilityChanged(ChannelOutboundBuffer.java:628) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.setUnwritable(ChannelOutboundBuffer.java:607) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.incrementPendingOutboundBytes(ChannelOutboundBuffer.java:177) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.addMessage(ChannelOutboundBuffer.java:129) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.write(AbstractChannel.java:882) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.write(DefaultChannelPipeline.java:1367) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:709) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:792) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:702) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:697) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.http.netty.HttpObjectEncoder.encodeChunkedContent(HttpObjectEncoder.java:402) ~[servicetalk-http-netty-0.42.18.jar:0.42.18]
	at io.servicetalk.http.netty.HttpObjectEncoder.write(HttpObjectEncoder.java:206) ~[servicetalk-http-netty-0.42.18.jar:0.42.18]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:709) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:792) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:702) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline.write(DefaultChannelPipeline.java:1015) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel.write(AbstractChannel.java:301) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber$AllWritesPromise.writeNext(WriteStreamSubscriber.java:413) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber.doWrite(WriteStreamSubscriber.java:174) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber.lambda$onNext$2(WriteStreamSubscriber.java:164) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.kqueue.KQueueEventLoop.run(KQueueEventLoop.java:295) ~[netty-transport-classes-kqueue-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: io.netty.channel.unix.Errors$NativeIoException: writevAddresses(..) failed: Broken pipe
[DEBUG] 2023-03-06 08:29:41.850 [servicetalk-global-io-executor-1-3] NettyHttpServer$ErrorLoggingHttpSubscriber - [id: 0xb4c156ba, L:/[0:0:0:0:0:0:0:1%0]:9000 ! R:/[0:0:0:0:0:0:0:1%0]:62798] Unexpected error received, closing HTTP/1.1 connection due to:
io.netty.channel.socket.ChannelOutputShutdownException: Channel output shutdown
	at io.netty.channel.AbstractChannel$AbstractUnsafe.shutdownOutput(AbstractChannel.java:650) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.handleWriteError(AbstractChannel.java:953) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.flush0(AbstractChannel.java:933) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.kqueue.AbstractKQueueChannel$AbstractKQueueUnsafe.flush0(AbstractKQueueChannel.java:507) ~[netty-transport-classes-kqueue-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.flush(AbstractChannel.java:895) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.flush(DefaultChannelPipeline.java:1372) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush0(AbstractChannelHandlerContext.java:750) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush(AbstractChannelHandlerContext.java:742) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.flush(AbstractChannelHandlerContext.java:728) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.transport.netty.internal.DefaultNettyConnection$NettyToStChannelInboundHandler.channelWritabilityChanged(DefaultNettyConnection.java:815) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:441) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:428) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelWritabilityChanged(AbstractChannelHandlerContext.java:421) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelWritabilityChanged(DefaultChannelPipeline.java:1433) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:441) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:428) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelWritabilityChanged(DefaultChannelPipeline.java:931) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.fireChannelWritabilityChanged(ChannelOutboundBuffer.java:628) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.setUnwritable(ChannelOutboundBuffer.java:607) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.incrementPendingOutboundBytes(ChannelOutboundBuffer.java:177) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.addMessage(ChannelOutboundBuffer.java:129) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.write(AbstractChannel.java:882) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.write(DefaultChannelPipeline.java:1367) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:709) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:792) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:702) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:697) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.http.netty.HttpObjectEncoder.encodeChunkedContent(HttpObjectEncoder.java:402) ~[servicetalk-http-netty-0.42.18.jar:0.42.18]
	at io.servicetalk.http.netty.HttpObjectEncoder.write(HttpObjectEncoder.java:206) ~[servicetalk-http-netty-0.42.18.jar:0.42.18]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:709) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:792) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:702) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline.write(DefaultChannelPipeline.java:1015) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel.write(AbstractChannel.java:301) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber$AllWritesPromise.writeNext(WriteStreamSubscriber.java:413) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber.doWrite(WriteStreamSubscriber.java:174) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber.lambda$onNext$2(WriteStreamSubscriber.java:164) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.kqueue.KQueueEventLoop.run(KQueueEventLoop.java:295) ~[netty-transport-classes-kqueue-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: io.netty.channel.unix.Errors$NativeIoException: writevAddresses(..) failed: Broken pipe
[DEBUG] 2023-03-06 08:29:41.850 [servicetalk-global-io-executor-1-7] NettyHttpServer$ErrorLoggingHttpSubscriber - [id: 0xbe516ca7, L:/[0:0:0:0:0:0:0:1%0]:9000 ! R:/[0:0:0:0:0:0:0:1%0]:62803] Unexpected error received, closing HTTP/1.1 connection due to:
io.netty.channel.socket.ChannelOutputShutdownException: Channel output shutdown
	at io.netty.channel.AbstractChannel$AbstractUnsafe.shutdownOutput(AbstractChannel.java:650) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.handleWriteError(AbstractChannel.java:953) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.flush0(AbstractChannel.java:933) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.kqueue.AbstractKQueueChannel$AbstractKQueueUnsafe.flush0(AbstractKQueueChannel.java:507) ~[netty-transport-classes-kqueue-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.flush(AbstractChannel.java:895) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.flush(DefaultChannelPipeline.java:1372) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush0(AbstractChannelHandlerContext.java:750) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush(AbstractChannelHandlerContext.java:742) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.flush(AbstractChannelHandlerContext.java:728) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.transport.netty.internal.DefaultNettyConnection$NettyToStChannelInboundHandler.channelWritabilityChanged(DefaultNettyConnection.java:815) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:441) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:428) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelWritabilityChanged(AbstractChannelHandlerContext.java:421) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelWritabilityChanged(DefaultChannelPipeline.java:1433) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:441) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:428) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelWritabilityChanged(DefaultChannelPipeline.java:931) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.fireChannelWritabilityChanged(ChannelOutboundBuffer.java:628) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.setUnwritable(ChannelOutboundBuffer.java:607) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.incrementPendingOutboundBytes(ChannelOutboundBuffer.java:177) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.addMessage(ChannelOutboundBuffer.java:129) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.write(AbstractChannel.java:882) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.write(DefaultChannelPipeline.java:1367) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:709) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:792) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:702) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:697) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.http.netty.HttpObjectEncoder.encodeChunkedContent(HttpObjectEncoder.java:402) ~[servicetalk-http-netty-0.42.18.jar:0.42.18]
	at io.servicetalk.http.netty.HttpObjectEncoder.write(HttpObjectEncoder.java:206) ~[servicetalk-http-netty-0.42.18.jar:0.42.18]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:709) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:792) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:702) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline.write(DefaultChannelPipeline.java:1015) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel.write(AbstractChannel.java:301) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber$AllWritesPromise.writeNext(WriteStreamSubscriber.java:413) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber.doWrite(WriteStreamSubscriber.java:174) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber.lambda$onNext$2(WriteStreamSubscriber.java:164) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.kqueue.KQueueEventLoop.run(KQueueEventLoop.java:295) ~[netty-transport-classes-kqueue-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: io.netty.channel.unix.Errors$NativeIoException: writevAddresses(..) failed: Broken pipe
[DEBUG] 2023-03-06 08:29:41.850 [servicetalk-global-io-executor-1-5] NettyHttpServer$ErrorLoggingHttpSubscriber - [id: 0xff27e109, L:/[0:0:0:0:0:0:0:1%0]:9000 ! R:/[0:0:0:0:0:0:0:1%0]:62801] Unexpected error received, closing HTTP/1.1 connection due to:
io.netty.channel.socket.ChannelOutputShutdownException: Channel output shutdown
	at io.netty.channel.AbstractChannel$AbstractUnsafe.shutdownOutput(AbstractChannel.java:650) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.handleWriteError(AbstractChannel.java:953) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.flush0(AbstractChannel.java:933) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.kqueue.AbstractKQueueChannel$AbstractKQueueUnsafe.flush0(AbstractKQueueChannel.java:507) ~[netty-transport-classes-kqueue-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.flush(AbstractChannel.java:895) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.flush(DefaultChannelPipeline.java:1372) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush0(AbstractChannelHandlerContext.java:750) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush(AbstractChannelHandlerContext.java:742) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.flush(AbstractChannelHandlerContext.java:728) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.transport.netty.internal.DefaultNettyConnection$NettyToStChannelInboundHandler.channelWritabilityChanged(DefaultNettyConnection.java:815) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:441) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:428) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelWritabilityChanged(AbstractChannelHandlerContext.java:421) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelWritabilityChanged(DefaultChannelPipeline.java:1433) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:441) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelWritabilityChanged(AbstractChannelHandlerContext.java:428) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelWritabilityChanged(DefaultChannelPipeline.java:931) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.fireChannelWritabilityChanged(ChannelOutboundBuffer.java:628) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.setUnwritable(ChannelOutboundBuffer.java:607) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.incrementPendingOutboundBytes(ChannelOutboundBuffer.java:177) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.ChannelOutboundBuffer.addMessage(ChannelOutboundBuffer.java:129) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.write(AbstractChannel.java:882) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.write(DefaultChannelPipeline.java:1367) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:709) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:792) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:702) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:697) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.http.netty.HttpObjectEncoder.encodeChunkedContent(HttpObjectEncoder.java:402) ~[servicetalk-http-netty-0.42.18.jar:0.42.18]
	at io.servicetalk.http.netty.HttpObjectEncoder.write(HttpObjectEncoder.java:206) ~[servicetalk-http-netty-0.42.18.jar:0.42.18]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:709) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:792) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:702) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.DefaultChannelPipeline.write(DefaultChannelPipeline.java:1015) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.AbstractChannel.write(AbstractChannel.java:301) ~[netty-transport-4.1.82.Final.jar:4.1.82.Final]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber$AllWritesPromise.writeNext(WriteStreamSubscriber.java:413) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber.doWrite(WriteStreamSubscriber.java:174) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.servicetalk.transport.netty.internal.WriteStreamSubscriber.lambda$onNext$2(WriteStreamSubscriber.java:164) ~[servicetalk-transport-netty-internal-0.42.18.jar:0.42.18]
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.channel.kqueue.KQueueEventLoop.run(KQueueEventLoop.java:295) ~[netty-transport-classes-kqueue-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.82.Final.jar:4.1.82.Final]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: io.netty.channel.unix.Errors$NativeIoException: writevAddresses(..) failed: Broken pipe
[DEBUG] 2023-03-06 08:29:41.853 [servicetalk-global-io-executor-1-3] NettyChannelPublisher - [id: 0xb4c156ba, L:/[0:0:0:0:0:0:0:1%0]:9000 ! R:/[0:0:0:0:0:0:0:1%0]:62798] Cancelling subscription
[DEBUG] 2023-03-06 08:29:41.853 [servicetalk-global-io-executor-1-5] NettyChannelPublisher - [id: 0xff27e109, L:/[0:0:0:0:0:0:0:1%0]:9000 ! R:/[0:0:0:0:0:0:0:1%0]:62801] Cancelling subscription
[DEBUG] 2023-03-06 08:29:41.853 [servicetalk-global-io-executor-1-7] NettyChannelPublisher - [id: 0xbe516ca7, L:/[0:0:0:0:0:0:0:1%0]:9000 ! R:/[0:0:0:0:0:0:0:1%0]:62803] Cancelling subscription
[DEBUG] 2023-03-06 08:30:04.825 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Pool stats (total=8, active=0, idle=8, waiting=0)
[DEBUG] 2023-03-06 08:30:04.826 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Fill pool skipped, pool has sufficient level or currently being filled (queueDepth=0).
[DEBUG] 2023-03-06 08:30:34.825 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Pool stats (total=8, active=0, idle=8, waiting=0)
[DEBUG] 2023-03-06 08:30:34.825 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Fill pool skipped, pool has sufficient level or currently being filled (queueDepth=0).
[DEBUG] 2023-03-06 08:31:04.829 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Pool stats (total=8, active=0, idle=8, waiting=0)
[DEBUG] 2023-03-06 08:31:04.829 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Fill pool skipped, pool has sufficient level or currently being filled (queueDepth=0).
[DEBUG] 2023-03-06 08:31:34.833 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Pool stats (total=8, active=0, idle=8, waiting=0)
[DEBUG] 2023-03-06 08:31:34.834 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Fill pool skipped, pool has sufficient level or currently being filled (queueDepth=0).

```

I've eventually been able to reproduce the error with the following commands:

```
baremaps map dev \
   --log-level DEBUG  \
  --database 'jdbc:postgresql://localhost:5432/baremaps?user=baremaps&password=baremaps' \
  --tileset 'tileset.json' \
  --style 'style.json'
```

Then, executing the following http request prints an error in the console:

```
curl http://localhost:9000/tiles/10/539/359.mvt
```

```
[ERROR] 2023-04-08 20:29:13.755 [servicetalk-global-executor-1-3] DevResources - Tile error
org.apache.baremaps.database.tile.TileStoreException: org.postgresql.util.PSQLException: ERROR: syntax error at end of input
  Position: 7
	at org.apache.baremaps.database.tile.PostgresTileStore.read(PostgresTileStore.java:122) ~[baremaps-core-0.7.2-SNAPSHOT.jar:?]
	at org.apache.baremaps.server.DevResources.getTile(DevResources.java:137) ~[baremaps-server-0.7.2-SNAPSHOT.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
```

--- a/baremaps-core/src/main/java/org/apache/baremaps/database/tile/PostgresTileStore.java
+++ b/baremaps-core/src/main/java/org/apache/baremaps/database/tile/PostgresTileStore.java
@@ -100,16 +100,17 @@ public class PostgresTileStore implements TileStore {
         Statement statement = connection.createStatement();
         ByteArrayOutputStream data = new ByteArrayOutputStream()) {
 
-      String sql = withQuery(tile);
-      logger.debug("Executing query: {}", sql);
-
       int length = 0;
-      try (GZIPOutputStream gzip = new GZIPOutputStream(data);
-          ResultSet resultSet = statement.executeQuery(sql)) {
-        while (resultSet.next()) {
-          byte[] bytes = resultSet.getBytes(1);
-          length += bytes.length;
-          gzip.write(bytes);
+      if (queries.stream().anyMatch(query -> zoomPredicate(query, tile.z()))) {
+        String sql = withQuery(tile);
+        logger.debug("Executing query: {}", sql);
+        try (GZIPOutputStream gzip = new GZIPOutputStream(data);
+            ResultSet resultSet = statement.executeQuery(sql)) {
+          while (resultSet.next()) {
+            byte[] bytes = resultSet.getBytes(1);
+            length += bytes.length;
+            gzip.write(bytes);
+          }
         }
       }
 

--- a/baremaps-core/src/main/java/org/apache/baremaps/database/tile/TileCache.java
+++ b/baremaps-core/src/main/java/org/apache/baremaps/database/tile/TileCache.java
@@ -53,7 +53,12 @@ public class TileCache implements TileStore {
   public ByteBuffer read(Tile tile) throws TileStoreException {
     return cache.get(tile, t -> {
       try {
-        return tileStore.read(t).duplicate();
+        var buffer = tileStore.read(t);
+        if (buffer == null) {
+          return null;
+        } else {
+          return buffer.duplicate();
+        }
       } catch (TileStoreException e) {
         logger.error("Unable to read the tile.", e);
         return null;

--- a/baremaps-core/src/main/java/org/apache/baremaps/vectortile/VectorTileUtils.java
+++ b/baremaps-core/src/main/java/org/apache/baremaps/vectortile/VectorTileUtils.java
@@ -92,6 +92,7 @@ public class VectorTileUtils {
         .encodeTile(vectorTile)
         .toByteString()
         .asReadOnlyByteBuffer();
+
   }
 
   /**

--- a/baremaps-server/src/main/java/org/apache/baremaps/server/ServerResources.java
+++ b/baremaps-server/src/main/java/org/apache/baremaps/server/ServerResources.java
@@ -86,9 +86,14 @@ public class ServerResources {
     try {
       ByteBuffer blob = tileStore.read(tile);
       if (blob != null) {
+        byte[] bytes = new byte[blob.remaining()];
+        blob.get(bytes);
         return Response.status(200) // lgtm [java/xss]
-            .header(ACCESS_CONTROL_ALLOW_ORIGIN, "*").header(CONTENT_TYPE, TILE_TYPE)
-            .header(CONTENT_ENCODING, TILE_ENCODING).entity(blob.array()).build();
+            .header(ACCESS_CONTROL_ALLOW_ORIGIN, "*")
+            .header(CONTENT_TYPE, TILE_TYPE)
+            .header(CONTENT_ENCODING, TILE_ENCODING)
+            .entity(bytes)
+            .build();
       } else {
         return Response.status(204).build();
       }

